locale/ar/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -314,7 +314,7 @@ http.createServer((request, response) => {
 
 رائع! الآن نستقر على تبسيط هذا وتذكر كائنات الطلب `request` هي تدقف قابل للقراءة
 [`ReadableStream`][] و كائنات الجواب `response` هي تدفق قابل للكتابة [`WritableStream`][].
-وهذا يعني أنه يمكننا إستخدام مَجْرىً لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده
+وهذا يعني أنه يمكننا إستخدام مَجْرىً لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده<!-- fix me -->
```

Comment:
```suggestion
وهذا يعني أنه يمكننا إستخدام [`pipe`][] لتوجيه البيانات من واحدة لأخرى. وهذا تماما مانريده
```

Reply:
Thanks! I'll rebase out my junk commit and leave your real fix 👍